AzureCalling/app/src/main/java/com/azure/samples/communication/calling/activities/CallActivity.java

```diff
@@ -375,12 +380,24 @@ private void openShareDialogue() {
     }
 
     private void setLayoutComponentState(
-            final boolean isMicrophoneMuted, final boolean isCameraOn, final boolean isCallHangUpOverLaid) {
+            final boolean isMicrophoneMuted, final boolean isCameraOn,
+            final boolean isCallHangUpOverLaid) {
         audioImageButton.setSelected(!isMicrophoneMuted);
         videoImageButton.setSelected(isCameraOn);
         callHangupOverlay.setVisibility(isCallHangUpOverLaid ? View.VISIBLE : View.INVISIBLE);
     }
 
+    private void openAudioDeviceList() {
+        if (audioDeviceSelectionPopupWindow == null) {
+            final AudioSessionManager audioSessionManager
+                    = ((AzureCalling) getApplicationContext()).getAudioSessionManager();
+            audioDeviceSelectionPopupWindow = new AudioDeviceSelectionPopupWindow(this, audioSessionManager);
+        } else {
```

Comment:
If `showAtLocation` is not called from the constructor, the `else` can be removed.

Reply:
addressed